README.md

```diff
@@ -25,11 +24,13 @@ To evaluate a model, (e.g. GPT-2) on NLU tasks (e.g. LAMBADA, HellaSwag), you ca
 
 ```bash
 python main.py \
-	--model gpt2 \
-	--device cuda:0 \
-	--tasks lambada,hellaswag
+	--model hf-causal \
+  --model_args pretrained=gpt2 \
+	--tasks mrpc
```

Comment:
I would like to see multiple tasks listed here, to show that the correct syntax is `task1,task2`